ci-operator/step-registry/cucushift/installer/check/azure/vnet/tag/cucushift-installer-check-azure-vnet-tag-ref.yaml

```diff
@@ -0,0 +1,22 @@
+ref:
+  as: cucushift-installer-check-azure-vnet-tag
+  from_image:
+   namespace: ocp
+   name: "4.14"
+   tag: upi-installer
+  commands: cucushift-installer-check-azure-vnet-tag-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: AZURE_VNET_TAGS
+    default: ""
+    documentation: |-
+      Format: "key1=value1 key2=value2"
+      Add customer tags on existing vnet to test shared tags added by installer is not overriden.
+  - name: EXIT_ON_INSTALLER_CHECK_FAIL
```

Comment:
If this ENV is not required, so we can remove it, too.

Reply:
yes, removed.